R/renv.R

```diff
@@ -0,0 +1,86 @@
+translateRenvToPackrat <- function(bundleDir) {
```

Comment:
What about materializing the manifest directly from the renv.lock?

Reply:
I thought we still needed the `packrat.lock` file because that's what connect uses?